x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_package_custom_extension/components/fleet_event_filters_card.tsx

```diff
@@ -0,0 +1,115 @@
+import { getEventFiltersListPath } from '../../../../../../common/routing';
+import { GetExceptionSummaryResponse } from '../../../../../../../../common/endpoint/types';
+import { PLUGIN_ID as FLEET_PLUGIN_ID } from '../../../../../../../../../fleet/common';
+import { MANAGEMENT_APP_ID } from '../../../../../../common/constants';
+import { useToasts } from '../../../../../../../common/lib/kibana';
+import { LinkWithIcon } from './link_with_icon';
+import { ExceptionItemsSummary } from './exception_items_summary';
+import { EventFiltersHttpService } from '../../../../../event_filters/service';
+
+export const FleetEventFiltersCard = memo<PackageCustomExtensionComponentProps>(({ pkgkey }) => {
+  const {
+    services: {
+      application: { getUrlForApp },
+      http,
+    },
+  } = useKibana<CoreStart & { application: ApplicationStart }>();
+  const toasts = useToasts();
+  const [stats, setStats] = useState<GetExceptionSummaryResponse | undefined>();
+  const eventFiltersListUrlPath = getEventFiltersListPath();
+  const eventFiltersApi = useMemo(() => new EventFiltersHttpService(http), [http]);
+
+  useEffect(() => {
+    const fetchStats = async () => {
+      try {
+        const summary = await eventFiltersApi.getSummary();
+        setStats(summary);
+      } catch (error) {
+        toasts.addDanger(
+          i18n.translate(
+            'xpack.securitySolution.endpoint.fleetCustomExtension.eventFiltersSummaryError',
+            {
+              defaultMessage: 'There was an error trying to fetch event filters stats: "{error}"',
+              values: { error },
+            }
+          )
+        );
+      }
+    };
+    fetchStats();
+  }, [eventFiltersApi, toasts]);
+
+  const eventFiltersRouteState = useMemo(() => {
+    const fleetPackageCustomUrlPath = `#${pagePathGetters.integration_details_custom({ pkgkey })}`;
+    return {
+      backButtonLabel: i18n.translate(
+        'xpack.securitySolution.endpoint.fleetCustomExtension.backButtonLabel',
+        { defaultMessage: 'Back to Endpoint Integration' }
+      ),
+      onBackButtonNavigateTo: [
+        FLEET_PLUGIN_ID,
+        {
+          path: fleetPackageCustomUrlPath,
+        },
+      ],
+      backButtonUrl: getUrlForApp(FLEET_PLUGIN_ID, {
+        path: fleetPackageCustomUrlPath,
+      }),
+    };
+  }, [getUrlForApp, pkgkey]);
+
+  return (
+    <EuiPanel paddingSize="l">
+      <EuiFlexGroup alignItems="baseline">
+        <EuiFlexItem grow={false}>
+          <EuiText>
+            <h4>
+              <FormattedMessage
+                id="xpack.securitySolution.endpoint.fleetCustomExtension.eventFiltersLabel"
+                defaultMessage="Event Filters"
+              />
+            </h4>
+          </EuiText>
+        </EuiFlexItem>
+        <EuiFlexItem style={{ alignItems: 'center', justifyContent: 'center' }}>
+          <ExceptionItemsSummary stats={stats} />
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <span>
```

Comment:
You should be able to use `<>` and `</>` here instead of `<span>` to reduce weight unless I'm missing something. Looks like you're just adding the `<span>` here to get two children? If so, [React fragments short version](https://reactjs.org/docs/fragments.html#short-syntax) are better.

Reply:
Yeah! You are totally right. This code was copy pasted from an old one. I'll change it since this `span` tag is not necessary.